bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -47,89 +48,87 @@ __program__ = "pycbc_pygrb_plot_chisq_veto"
+    
+    snr_type = opts.snr_type
     veto_type = opts.y_variable
-    if opts.use_sngl_ifo_veto:
-        veto_type += '_sngl'
 
-    # Inizialize the dictionary
+    # Initialize the dictionary
     data = {}
     data[snr_type] = None
     data[veto_type] = None
-    data['chisq_dof'] = None
-    data['bank_chisq_dof'] = None
-    data['cont_chisq_dof'] = None
+    data['dof'] = None
 
-    # Fill the dictionary in with required data
     if input_file:
-        if injections:
-            trigs_or_injs = ppu.load_injections(input_file, vetoes)
-        else:
-            trigs_or_injs = ppu.load_triggers(input_file, vetoes)
-        # Case with single SNR
-        if snr_type == 'snr_sngl':
-            data[snr_type] = trigs_or_injs.get_sngl_snr(opts.ifo)
-        # Coherent SNR
-        else:
-            data[snr_type] = numpy.asarray(trigs_or_injs.get_column('snr'))
-        # Chi-squared data
-        prefix_dict = {'standard': '', 'bank': 'bank_', 'auto': 'cont_'}
-        if '_sngl' not in veto_type:
-            data[veto_type] = \
-                numpy.asarray(trigs_or_injs.get_column(prefix_dict[veto_type]+'chisq'))
-        elif veto_type == 'standard_sngl':
-            data[veto_type] = trigs_or_injs.get_sngl_chisqs(ifos)
-        # Format single IFO chi-squared data (keep only data of interest)
-        if opts.use_sngl_ifo_veto:
-            data[veto_type] = format_single_chisqs(data[veto_type], ifos)[opts.ifo]
-        # Remove BestNR = 0 points when dealing with standard chi-square
-        if 'standard' in veto_type:
-            null_thresh = list(map(float, opts.null_snr_threshold.split(',')))
-            data['reweighted_snr'] = ppu.get_bestnrs(trigs_or_injs,
-                                                     q=opts.chisq_index,
-                                                     n=opts.chisq_nhigh,
-                                                     null_thresh=null_thresh,
-                                                     snr_threshold=opts.snr_threshold,
-                                                     sngl_snr_threshold=opts.sngl_snr_threshold,
-                                                     chisq_threshold=opts.newsnr_threshold,
-                                                     null_grad_thresh=opts.null_grad_thresh,
-                                                     null_grad_val=opts.null_grad_val)
-            data[snr_type] = data[snr_type][data['reweighted_snr'] != 0]
-            data[veto_type] = data[veto_type][data['reweighted_snr'] != 0]
-        # Gather data on degrees of freedom
-        for cs in ['', 'bank_', 'cont_']:
-            dof = cs+'chisq_dof'
-            data[dof] = numpy.unique(trigs_or_injs.get_column(dof))
+        trigs_or_injs = ppu.load_triggers(input_file, vetoes)
+        
+        num_trigs_or_injs = len(trigs_or_injs['network/end_time_gc'][:])
+        
+        if snr_type in ['coherent', 'null', 'reweighted']:
```

Comment:
Since you extended the scope of this executable (now the user can also plot vs null, reweighted, and coincident SNR), remember to update its dosctoring and the help message for `--snr-type` which is currently inconsistent with this line.